packages/components/src/date-time/index.js

```diff
@@ -28,6 +28,7 @@ function DateTimePicker(
 		onMonthPreviewed,
 		onChange,
 		events,
+		keepOpen,
```

Comment:
Should `keepOpen` be defined `true` as default? where else the `<DateTimePicker />` is rendered?

Reply:
I thought it was being used by other features, but it's just Storybook components. 😂 So it probably does make sense to default to true.

Reply:
I think it's ok keeping it as `false` as default, actually. So by default, it won't change anything. Is it correct?

Reply:
Yes, that's correct. If the `keepOpen` prop isn't passed to `DateTimePicker` or isn't `true` then there won't be any change to the popover behavior.